cfgov/v1/models/organisms.py

```diff
@@ -120,3 +120,17 @@ class Meta:
         icon = 'form'
         template = '_includes/organisms/table.html'
         label = 'Table'
+
+
+class ExpandableGroup(blocks.StructBlock):
+    heading = blocks.CharBlock(required=False)
+    body = blocks.RichTextBlock(required=False)
+
+    is_accordion = blocks.BooleanBlock(required=False)
+    has_rule = blocks.BooleanBlock(required=False)
+
+    expandables = blocks.ListBlock(molecules.Expandable())
+
+    class Meta:
+        icon = 'list-ul'
```

Comment:
nah this is just what icon it should use,  when adding the molecule in the streamfields